packages/@react-spectrum/menu/docs/MenuTrigger.mdx

```diff
@@ -121,6 +121,63 @@ import {Text} from '@react-spectrum/text';
 </MenuTrigger>
 ```
 
+## Unavilalbe Menu Item <VersionBadge version="alpha" style={{marginLeft: 4, verticalAlign: 'bottom'}} />
+
+ContextualHelpTrigger shows a user extra information about the state of an unavilalbe Menu Item.
+It provides context for the state and may point users to more information elsewhere.
+
+The ContextualHelpTrigger accepts exactly
+two children: the Item which triggers opening of the Dialog and the Dialog itself. The trigger must be
+the first child passed into the ContextualHelpTrigger and should be an Item. Similar to
+ContextualHelp, the layout of the Dialog is very diliberate. See [ContextualHelp](ContextualHelp.html#content) for further explanation.
+
+By default, a ContextualHelpTrigger's Dialog is opened by hovering or pressing the trigger element or activating
+it via the <Keyboard>Space</Keyboard>, <Keyboard>Enter</Keyboard>, or <Keyboard>Right Arrow</Keyboard> keys.
+Hovering another item or pressing the <Keyboard>Esc</Keyboard> key will close the Dialog and leave the Menu open.
+
+When Menu Items are unavailabe via a ContextualHelpTrigger, the Menu `onAction` and `onSelectionChange`
+callbacks will not occur.
+
+Setting the `isUnavailable` prop on the ContextualHelpTrigger disables a Menu Item.
```

Comment:
I would probably include more info on this use case (ability to have this be dynamic).